src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -31,33 +31,79 @@ redirects:
 
-Alerts allow you to define the conditions in your system you want to be notified about, letting you focus on the things you care about most. Applied intelligence is a machine-learning engine that notifies your team about potential problems by identifying anomalous behavior, correlating any relevant incidents, alerting your team of active issues, and providing a root cause analysis.
+New Relic's alerts and applied intelligence were designed for busy DevOps teams to end response fatigue: you can detect and diagnose incidents and prioritize them for immediate action. If you haven't already, If you haven't already, [sign up for your free account](https://newrelic.com/signup) to [start receiving alerts](https://one.newrelic.com/alerts-ai) about your data right away!
 
-Alerts and applied intelligence were created for busy DevOps teams: they reduce noise and make sure you never miss an issue. 
+<img
+  title="Alerts and AI homepage"
+  alt="A screenshot outlining the alerts and ai UI"
+  src={alertsAlertsandAiOverviewPage}
+/>
 
-## How it all works 
+## Navigate the alerts and applied intelligence UI [#get-started-with-alerts]
 
-Alerts and applied intelligence are a suite of tools and features that proactively notify you when things go wrong right when the problems begin.
+In the [New Relic UI](https://one.newrelic.com/alerts-ai), alerts and applied intelligence are divided into several sections, each unlocking different features. Check out the section explanations below to learn how these features can meet your needs.
 
-### Detect and diagnose potential problems in your system [#detect-intro]
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="Analyze"
+    title="Analyze"
+  >
+Use the analyze features to scan for critical problems affecting your systems, via alerts incidents, issues, and anomalies.
 
-New Relic never stops observing your telemetry data and we created alerts so your team can decide when you want to be notified and how.  
+* **[Overview](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page)**: Find your most recent open, closed, and muted incidents.
+* **Issues & activity**: Review your most recent [issues](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#issues), [incidents](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#incidents), and [anomalies](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#anomalies).
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="Detect"
+    title="Detect"
+  >
+Define problems in your system you want to be notified about with alert conditions.
+
```

Comment:
I feel like we are missing two links here to better align it with the UI screenshot above, that is Anomaly detection and Alert coverage gaps.